tools/tkn-results/internal/config/config.go

```diff
@@ -55,19 +56,20 @@ func init() {
 	viper.SetConfigName("results")
 	viper.SetConfigType("yaml")
 	viper.AddConfigPath("$HOME/.config/tkn")
-	pflag.StringP("addr", "a", "", "Result API server address")
-	pflag.StringP("authtoken", "t", "", "authorization bearer token to use for authenticated requests")
-	pflag.Parse()
+	err := setConfig()
+	if err != nil {
+		fmt.Println("error setting up flags and config", err)
+	}
 }
 
-func GetConfig() (*Config, error) {
+func setConfig() error {
 	if err := viper.BindPFlags(pflag.CommandLine); err != nil {
-		return nil, err
+		fmt.Println(err)
```

Comment:
Why don't we return err here?

Reply:
I forgot to update this. It should be fmt.Fatal

Reply:
Actually, we should only return err here and do log.Fatal in init function.